code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -366,3 +366,12 @@
 		return MONKEY_HEIGHT_MEDIUM
 
 	return mob_height
+
+///Gets organ slot by name
+/mob/living/carbon/human/proc/get_organ_slot(var/string)
```

Comment:
For both this and remove it's not actually getting or removing the slot, right? It's returning the organ itself.